product_code_uniqueness/models/product_product.py

```
@@ -46,7 +51,7 @@ def _check_default_code(self):
                 )
                 if matches > 1:
                     raise ValidationError(
-                        _(
```

there is no longer `_` importable translation?

---

@oerp-odoo env needs to be added for translations [[source](https://github.com/odoo/odoo/pull/174844)]

---

Nice, thats actually better approach. Previously there were cases when translations would not work and it would silently ignore it, for this same reason as it could not identify `env` (e.g. using translations in functions without `env`).